setup.sh

```diff
@@ -191,6 +198,7 @@ fi
 
 cp $INIT_DIR/python/* $INST_PATH/bin
 cp $INIT_DIR/R/* $INST_PATH/bin/
+cp $INIT_DIR/perl/* $INST_PATH/bin
```

Comment:
Make sure perl and python have appropriate permissions once deployed.  World execute is not maintained via git, only user:

```
chmod +x $INST_PATH/bin/*.py $INST_PATH/bin/*.pl
```

Reply:
Still an issue